test.sh

```diff
@@ -69,37 +76,20 @@ UUID=${NEW_UUID%-*}
 tag_name="${NODE_NAME:-master}"
 
 sed -i "s#ES_SERVER#$ES_SERVER#g" tests/test_crs/*
-sed -i "s/sql-server/sql-server-$UUID/g" tests/mssql.yaml tests/test_crs/valid_hammerdb.yaml tests/test_hammerdb.sh
-sed -i "s/benchmarks.ripsaw.cloudbulldozer.io/benchmarks-$UUID.ripsaw.cloudbulldozer.io/g" resources/crds/ripsaw_v1alpha1_ripsaw_crd.yaml
-sed -i "s/kind: Benchmark/kind: Benchmark-$UUID/g" resources/crds/ripsaw_v1alpha1_ripsaw_crd.yaml
-sed -i "s/listKind: BenchmarkList/listKind: BenchmarkList-$UUID/g" resources/crds/ripsaw_v1alpha1_ripsaw_crd.yaml
-sed -i "s/plural: benchmarks/plural: benchmarks-$UUID/g" resources/crds/ripsaw_v1alpha1_ripsaw_crd.yaml
-sed -i "s/singular: benchmark/singular: benchmark-$UUID/g" resources/crds/ripsaw_v1alpha1_ripsaw_crd.yaml
-sed -i "s/benchmarks/benchmarks-$UUID/g" tests/common.sh
-sed -i "s/kind: Benchmark/kind: Benchmark-$UUID/g" tests/test_crs/*.yaml
-sed -i "s/kind: Benchmark/kind: Benchmark-$UUID/g" playbook.yml
-sed -i "s/kind: Benchmark/kind: Benchmark-$UUID/g" watches.yaml
-sed -i "s/backpack_role/backpack_role-$UUID/g" resources/backpack_role.yaml
-sed -i "s/my-ripsaw/my-ripsaw-$UUID-test-fiod/g" resources/kernel-cache-drop-daemonset.yaml
-grep -Rl "kind: Benchmark" roles/ | xargs sed -i "s/kind: Benchmark/kind: Benchmark-$UUID/g"
-sed -i "s|          image: quay.io/benchmark-operator/benchmark-operator:master*|          image: $image_location/$image_account/benchmark-operator:$tag_name # |" resources/operator.yaml
-
 cp -pr * gold/
 
 # Create individual directories for each test
 for ci_dir in `cat tests/my_tests`
 do
   mkdir $ci_dir
   cp -pr gold/* $ci_dir/
-  cd $ci_dir/
-  # Edit the namespaces so we can run in parallel
-  sed -i "s/my-ripsaw/my-ripsaw-$UUID-$ci_dir/g" `grep -Rl my-ripsaw`
-  sed -i "s/benchmark-operator-role/benchmark-operator-role-$UUID-$ci_dir/g" `grep -Rl benchmark-operator-role`
-  cd ..
 done
 
-# Update the operator image
-update_operator_image
+delete_operator || true
+oc delete namespace benchmark-operator --ignore-not-found
+kubectl delete benchmarks -n benchmark-operator --ignore-not-found --all
```

Comment:
Maybe we can skip this line?, deleting the namespace benchmark-operator should remove all objects in that namespace

Reply:
added